agent/structs/acl.go

```diff
@@ -72,10 +72,6 @@ session_prefix "" {
 	policy = "write"
 }` + EnterpriseACLPolicyGlobalManagement
 
-	// This is the policy ID for anonymous access. This is configurable by the
-	// user.
-	ACLTokenAnonymousID = "00000000-0000-0000-0000-000000000002"
```

Comment:
This PR only migrates the anonymous token definition for ACLs.. I don't disagree with this at all but just want to point out that now we have ACL definitions in two places, `acl/` and `structs/`.. Is it worth adding a note in the code indicating that we intend to move the rest of the ACL defs to the `acl/` package at some point?

Reply:
Yeah it was originally an initiative started in core before Mark A left; the intent had always been to migrate things over. I'll add a todo

Reply:
As I was looking over the file I realized that there are some import cycles that might make a general migration difficult. Unsure if a general TODO would be helpful (or if it will grow stale and unaddressed). I think just an ad-hoc refactoring over time might be better?

Reply:
Ya, I'm fine with whatever you think is best. I suspect that we will be thinking about and revisiting the ACLs system over the coming months and this is not likely to fall off our radar.